src/components/WwwFrame/WwwPage.vue

```diff
@@ -13,7 +13,9 @@
 		<the-footer
 			:theme="footerTheme"
 		/>
-		<the-basket-bar />
+		<the-basket-bar
+			v-if="activeOnPage"
+		/>
```

Comment:
These changes were added to hide the existing "basket" button on small screens that is shown site wide. We're showing the new LendCta component only on the /lend-beta page.